CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [5.0.0] - 2021-02-02
+
+### Changed
+
+- **(SEMVER-MAJOR)** Enable `semi` in base config ([#101](https://github.com/MetaMask/eslint-config/pull/101))
+  - This may just be fixable with `eslint . --fix`, but we consider it breaking because it's a major stylistic change.
```

Comment:
I mean, anything that caused the linter to fail where it would previously have passed should be considered breaking. So it's not entirely because this is a major stylistic change.

Reply:
Similarly, the change to function spacing is also breaking

Reply:
Sure!